libs/blocks/language-selector/language-selector.js

```diff
@@ -327,8 +382,10 @@ function setupDropdownEvents({
     filteredLanguages = doRenderLanguages(e.target.value);
   }, 200);
   searchInput.addEventListener('input', debouncedInput);
-
   searchInput.addEventListener('keydown', (e) => {
+    if (e.key === ' ') {
```

Comment:
Not strictly related to these code changes, but I did notice that one cannot tab through the languages; navigation can only be done with arrow keys. Has this been approved from an accessibility perspective?